lib/scraped/json.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'json'
+
+module Scraped
+  class JSON < Scraped::Document
```

Comment:
I think this should just be `Document`, rather than `Scraped::Document`, to be consistent with other class definitions.